ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicaThread.java

```diff
@@ -99,6 +101,44 @@ public int compare(RemoteReplicaInfo r1, RemoteReplicaInfo r2) {
     }
   }
 
+  @Override
+  public void run() {
+    try {
+      int delay = new Random().nextInt(900) + 1;
+      logger.info("Starting replica thread {} in {} seconds", Thread.currentThread().getName(), delay);
+      Thread.sleep(TimeUnit.SECONDS.toMillis(delay));
```

Comment:
let's use a configuration here. just in case we want to change it in the future.

Reply:
done